tests/test_transform.py

```diff
@@ -64,10 +65,14 @@ def download_file(unittest_adapter, output_dir):
     function, use assert to check if the file is downloaded.
 
     """
+    if os.path.isdir(output_dir):
+        rmtree(output_dir)
+
+    os.makedirs(output_dir)
```

Comment:
This replaces the `HarmonyAdapter.prepare_output_dir` method that wasn't being used anywhere else.